core/src/main/resources/catalog.proto

```diff
@@ -1,6 +1,6 @@
 syntax = "proto3";
 option go_package = "github.com/treevese/lakefs/catalog";
-option java_package = "io.treeverse.catalog";
+option java_package = "io.treeverse";
```

Comment:
This doesn't match the `go_package`; why do this?  (If anything, it should have been a deeper namespace: imagine Treeverse in 5 years, after it buys Digital Ocean, Azure, and AWS: having these at the top-level of `io.treeverse` will be somewhat embarrassing...)